src/Handler/Bucket/Link/LinkBucketHandler.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\BigQuery\Handler\Bucket\Link;
+
+use Google\Cloud\BigQuery\AnalyticsHub\V1\DestinationDataset;
+use Google\Cloud\BigQuery\AnalyticsHub\V1\DestinationDatasetReference;
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\BigQuery\GCPClientManager;
+use Keboola\StorageDriver\BigQuery\NameGenerator;
+use Keboola\StorageDriver\Command\Bucket\LinkBucketCommand;
+use Keboola\StorageDriver\Command\Bucket\LinkedBucketResponse;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+
+final class LinkBucketHandler implements DriverCommandHandlerInterface
+{
+    public GCPClientManager $clientManager;
+
+    public function __construct(GCPClientManager $clientManager)
+    {
+        $this->clientManager = $clientManager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials, // main credentials
+        Message $command, // linked bucket
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof LinkBucketCommand);
+
+        assert($command->getStackPrefix() !== '');
```

Comment:
tady by to imho chtelo nejake assertovske hlasky . neco jako tady https://github.com/keboola/php-storage-driver-teradata/pull/45/files#diff-45fe9e223a136172e8f448cdc17a35fad984451466ce61a72faae4362e2f3d20R90

Reply:
To som tam vlastne schvalne nepisal pretoze ked to failne ten assert tak to vyhodi hlasku `Fatal error: Uncaught AssertionError: assert($command->getStackPrefix()  !== '')`  co mi pride ako dostacujuca informacia

Reply:
tam to naviac kontrolue v tom odkaze ze je to numeric

Reply:
v tom odkaze jsem chtel ukazat to, ze ta hlaska ma cestu toho parametru (z pohledu protobuf) a je pochopitelna pro endusera (kdyz se na to budeme divat jako na API). To co rikas ty je "pochopime z logu proc to skoncilo 500".